format/spec.md

```diff
@@ -1021,7 +1023,11 @@ Lists must use the [3-level representation](https://github.com/apache/parquet-fo
 | **`struct`**       | `group`                                                            |                                             |                                                                |
 | **`list`**         | `3-level list`                                                     | `LIST`                                      | See Parquet docs for 3-level representation.                   |
 | **`map`**          | `3-level map`                                                      | `MAP`                                       | See Parquet docs for 3-level representation.                   |
+| **`geometry`**     | `binary`                                                           | `GEOMETRY`                                  | WKB format, see [Appendix G](#appendix-g-geospatial-notes). Logical type annotation optional for supported Parquet format versions [1]. |
+
+Notes:
 
+1. [https://github.com/apache/parquet-format/pull/240](https://github.com/apache/parquet-format/pull/240))
```

Comment:
I'd prefer not to reference a PR.

Reply:
Removed these, can add the logical types once the pr is merged in Parquet